docs/user_guide/citing.md

```diff
@@ -0,0 +1,39 @@
+# How to cite SuperDARN
+
+Super Dual Auroral Radar Network (SuperDARN) is a made up of 36 radars 20 institutions, to cite general SuperDARN you can use: 
+
+- Greenwald, R.A., Baker, K.B., Dudeney, J.R. et al. Space Sci Rev (1995) 71: 761. [doi:10.1007/BF00751350](https://doi.org/10.1007/BF00751350)
+
+For the general achievements of the SuperDARN Network, you can read these papers: 
+
+- Chisham, G., Lester, M., Milan, S.E. et al. A decade of the Super Dual Auroral Radar Network (SuperDARN): scientific achievements, new techniques and future directions. Surv Geophys 28, 33–109 (2007) [doi:10.1007/s10712-007-9017-8](https://link.springer.com/article/10.1007/s10712-007-9017-8)
+- Nishitani, N., Ruohoniemi, J.M., Lester, M. et al. Review of the accomplishments of mid-latitude Super Dual Auroral Radar Network (SuperDARN) HF radars. Prog Earth Planet Sci 6, 27 (2019) [doi:10.1186/s40645-019-0270-5](https://progearthplanetsci.springeropen.com/articles/10.1186/s40645-019-0270-5)
+
+During your study, if using data from individual radars only, please contact the Principal Investigator (PI) of that radar about potential co-authorship. 
+A list of radars, institutions, and their PI's information can be found [here](http://vt.superdarn.org/tiki-index.php?page=Radar+Overview).  
+
+## DOI
+
+Currently SuperDARN in the process of placing DOI's on their data set. In the meantime, please use any local available services to DOI your data set. 
+Possible services:
+
+  - [FRDR](https://www.frdr.ca/repo/): Canadian resource 
+  - [zenodo](https://help.zenodo.org/features/) 
+
+# Citing RST
+
+Click on DOI badge:
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3994968.svg)](https://doi.org/10.5281/zenodo.3994968)
+
+Scroll to the bottom and copy *Share Cite as* text or clicking on the preferred *Export* type. 
+
+## Acknowledgments 
+
+RST is also built on various scientific algorithms and libraries: 
+
+- [AACGM and AACGMv2](http://superdarn.thayer.dartmouth.edu/aacgm.html)
+- IGRF
+- Chisham Virtual Hieght Model 
+- Dr. Evan Thomas Statistical Convection Map models 
+- Dr. Shepherd Elevation Angle algorithm 
+-
```

Comment:
Perhaps we should finalise the procedure for acknowledging these libraries before including this information here? We also need to make a more comprehensive list of libraries (it will be quite long). If we're keen on including a preliminary list now, we should provide links to the published papers (I can help with that).

Reply:
So you want to delete this section for now?

Reply:
ok, I suggest that we can delete this for now and revisit it when we've made more progress with the authorship/acknowledgements. 

```suggestion
```